src/stampy_nlp/routes.py

```diff
@@ -34,6 +34,16 @@ def check_auth(username, password):
     return AUTH_PASSWORD and username == 'stampy' and password == AUTH_PASSWORD
 
 
+def get_query(request):
+    if request.method == 'GET':
+        query = request.args.get('query', DEFAULT_QUERY)
```

Comment:
this is what was causing the problem - when `query` is set to e.g. 'who am I?' everything is fine. When `query` is missing it's also fine, as it then uses the default query. The issue is when `query` is set to an empty string, as then `request.args.get('query')` and its ilk will happily return the empty string. Which is correct, as the key exists, after all